tests/Unit/DataStructures/DataBox/Test_DataBoxDocumentation.cpp

```diff
@@ -0,0 +1,527 @@
+  const double mass = mass_compute(box);
+  const double acceleration = acceleration_compute(box);
+  return mass * acceleration;
+}
+/// [std_map_databox_force_compute]
+
+double std_map_databox_2(){
+/// [std_map_databox_small_program_2]
+// Set up variables:
+const double velocity = 4.0;
+const double radius = 2.0;
+const double density = 0.5;
+const double volume = 10.0;
+
+// Set up databox:
+std::map<std::string, double> naive_databox;
+naive_databox["Velocity"] = velocity;
+naive_databox["Radius"] = radius;
+naive_databox["Density"] = density;
+naive_databox["Volume"] = volume;
+
+// Use variables:
+return force_compute(naive_databox);
+/// [std_map_databox_small_program_2]
+}
+
+bool std_tuple_databox_example(){
+/// [std_tuple_databox_1]
+std::tuple<double, size_t, bool> sophomore_databox =
+  std::make_tuple(1.2, 8, true);
+/// [std_tuple_databox_1]
+
+/// [std_tuple_databox_2]
+const bool bool_quantity = std::get<bool>(sophomore_databox);
+// value obtained is `true`
+/// [std_tuple_databox_2]
+return bool_quantity;
+}
+
+namespace sophomore {
+/// [std_tuple_tags]
+struct Velocity{};
+struct Radius{};
+struct Density{};
+struct Volume{};
+/// [std_tuple_tags]
+
+double std_tuple_databox_1(){
+/// [std_tuple_small_program_1]
+std::tuple<std::pair<Velocity,double>,
+           std::pair<Radius, double>, std::pair<Density, double>,
+           std::pair<Volume, double>> sophomore_databox =
+  std::make_tuple(std::make_pair(Velocity{}, 4.0),
+                  std::make_pair(Radius{}, 2.0), std::make_pair(Density{}, 0.5),
+                  std::make_pair(Volume{}, 10.0));
+/// [std_tuple_small_program_1]
+  return std::get<std::pair<Density, double>>(sophomore_databox).second *
+         std::get<std::pair<Volume, double>>(sophomore_databox).second *
+         std::get<std::pair<Velocity, double>>(sophomore_databox).second *
+         std::get<std::pair<Velocity, double>>(sophomore_databox).second /
+         std::get<std::pair<Radius, double>>(sophomore_databox).second;
+}
+
+/// [std_tuple_mass_compute]
+template<typename... Pairs>
+double mass_compute(const std::tuple<Pairs...>& box) noexcept {
+  return std::get<std::pair<Density, double>>(box).second *
+         std::get<std::pair<Volume, double>>(box).second;
+}
+/// [std_tuple_mass_compute]
+
+/// [std_tuple_acceleration_compute]
+template<typename... Pairs>
+double acceleration_compute(const std::tuple<Pairs...>& box) noexcept {
+  return std::get<std::pair<Velocity, double>>(box).second *
+         std::get<std::pair<Velocity, double>>(box).second /
+         std::get<std::pair<Radius, double>>(box).second;
+}
+/// [std_tuple_acceleration_compute]
+
+/// [std_tuple_force_compute]
+template<typename... Pairs>
+double force_compute(const std::tuple<Pairs...>& box) noexcept {
+  const double mass = mass_compute(box);
+  const double acceleration = acceleration_compute(box);
+  return mass * acceleration;
+}
+/// [std_tuple_force_compute]
+
+double std_tuple_databox_2(){
```

Comment:
there's general inconsistency in this file with `noexcept` and spaces between `(){`. Please be mindful of this in the future :)